api/pkg/apis/projectcalico/v3/constants.go

```diff
@@ -28,6 +28,14 @@ const (
 	// AllNames is used for List or Watch queries to wildcard the name.
 	AllNames = ""
 
+	// Label used to denote the Kind. This is added to network sets by Calico and may be used for
+	// label matches by Policy selectors.
+	LabelKind = "projectcalico.org/kind"
```

Comment:
Don't think this is used?  Suggest upstreaming this feature since it sounds useful/inconsistent to only have it in enterprise.

Reply:
I removed it in this PR. I am going to use another PR to move this change and the related ones from enterprise to open source.